helix-view/src/graphics.rs

```diff
@@ -327,17 +365,16 @@ bitflags! {
     ///
     /// let m = Modifier::BOLD | Modifier::ITALIC;
     /// ```
-    #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+    #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize), serde(rename_all = "kebab-case"))]
```

Comment:
Does this end up having an effect? I still see the snake_case version working because of the FromStr implementation below

Reply:
This is leftover from the original PR which added a bunch of modifiers for underline styles (and probably didn't have the intended effect anyway as you noted). I have just reset the `Modifiers` enum back to master and just removed the `UNDERLINE` variant (as thats now covered by `underline-style`).

I should probably just completely start from scratch when I pickup a PR and do a major refactor in the future (same thing happened in #3890)